src/ValueObjects/Address.php

```diff
@@ -24,92 +24,57 @@ class Address
      */
     protected $streetAddress;
 
-    /**
-     * Address constructor.
-     * @param string $addressCountry
-     * @param string $addressLocality
-     * @param string $postalCode
-     * @param string $streetAddress
-     */
     public function __construct(
-        $addressCountry = null,
-        $addressLocality = null,
-        $postalCode = null,
-        $streetAddress = null
+        string $addressCountry = null,
+        string $addressLocality = null,
+        string $postalCode = null,
+        string $streetAddress = null
```

Comment:
should these be `?string` instead?

Reply:
If you specify the default value of an argument as `null` it's also nullable regardless of if you use `string` or `?string`, but I'll update it for clarity 👍

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/db1d1283aa72d0ee0957724fc40c8bd6754a7eb8